### PR TITLE
Add zh_CN locale

### PR DESCRIPTION
**Pull request by yfdyh000**
Translated via https://lusito.github.io/web-ext-translator/

### CODE REVIEW

File: src/_locales/zh_CN/messages.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
+    "extensionLocale": {
```
Comment (asamuzaK) [inline src/_locales/zh_CN/messages.json]: Some nits for code styling.
2 spaces for indent please.

File: src/_locales/zh_CN/messages.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        "message": "zh-CN",
+        "hash": "e2d59e95a3a41434b5a59d9ac0897d74"
```
Comment (asamuzaK) [inline src/_locales/zh_CN/messages.json]: Remove "hash" field please.

File: src/_locales/zh_CN/messages.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            "sns": { "content": "$1", "example": "Twitter" }
+        },
+        "hash": "4bc83da8495cbeb81c4d80a3bfd0c5b0"
+    },
+    "snsNotSelected": {
+        "message": "未选择社交网络。",
+        "hash": "71a094c5e213a982a6443faa3ed7a2fc"
+    },
+
+    "__WET_LOCALE__": { "message": "zh-CN" }
```
Comment (asamuzaK) [inline src/_locales/zh_CN/messages.json]: Remove this field too.

**Review by asamuzaK — CHANGES_REQUESTED** [review]

File: src/_locales/zh_CN/messages.json [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        "hash": "30c27f028a6de9ba1f637730375bfffa"
+    },
+    "extensionShortName": {
+        "message": "分享",
+        "hash": "7701b2436f566ea956d2cd9bdb6bd075"
+    },
+    "optionsEnableSnsDetail": {
+        "message": "启用分享到 $SNS$。",
+        "placeholders": {
+            "sns": { "content": "$1", "example": "Twitter" }
```
Comment (asamuzaK) [inline src/_locales/zh_CN/messages.json]: New line after bracket and also comma please.

**Comment by yfdyh000** [comment]
Cleaned up, although I think it is not very valuable.

**Review by asamuzaK — APPROVED** [review]

**Comment by asamuzaK** [comment]
Thanks!

**Pull request merged**